packages/web/config/chain-infos.ts

```diff
@@ -76,7 +76,8 @@ const chainInfos = (
           coinDenom: "LUNC",
           coinMinimalDenom: "uluna",
           coinDecimals: 6,
-          coinGeckoId: "terra-luna",
+          //coinGeckoId: "terra-luna",
+          coinGeckoId: "pool:uluna",
```

Comment:
This is not going to get confused b/w LUNA vs LUNC right?

Reply:
Is there a way to rename this pool to `pool:ulunc` by any chance?

Reply:
yes, makes sense. Done